plugins/org.orbisgis.h2gis/plugin.xml

```diff
@@ -9,7 +9,7 @@
         <datasource
                 class="org.orbisgis.h2gis.H2GIS200DataSourceProvider"
                 description="%datasource.h2gis.description"
-                id="h2gis 2.0.0"
+                id="h2gis"
```

Comment:
Can you change the id to something like 'h2gis snapshot' or 'h2Gis beta' in order to avoid conflict with the release datasource which could be named 'h2gis'